pom.xml

```diff
@@ -209,6 +209,12 @@
       <version>2.5</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Well, Equalsverifier checks equals and hashCode methods soroly.
Therefore I considered to use it.